metricbeat/module/http/_meta/fields.yml

```diff
@@ -5,6 +5,7 @@
   release: ga
   settings: ["ssl", "http"]
   fields:
+    # Is this all ECS compatible?
     - name: http
```

Comment:
Metricbeat collects body content (not size)? I'm surprised, I thought Mb polled services to get numeric data on a schedule. Is this a leftover, or it's actually being used by Metricbeat?

If this is actually being used:

- http.request.body => http.request.body.content
- http.response.body => http.response.body.content

We have not explored the headers yet, so I'm not sure what form it would take. I'd love to see what the data looks like. I assume we'll want to leave room for metadata, as we do for body. E.g. `.headers.count`

Reply:
I opened a PR for this here. It is used. There is a config to enable body fetching. https://github.com/elastic/beats/pull/10315